### PR TITLE
Add jit newValue helper

### PR DESCRIPTION
**Pull request by tajila**
Add jit newValue helper

This PR depends on: https://github.com/eclipse/openj9/pull/8110

Signed-off-by: Tobi Ajila <atobia@ca.ibm.com>

### CODE REVIEW

**Comment by tajila** [comment]
@gacholio @fjeremic Please review these changes

File: compiler/ras/Debug.cpp [hunk 1]
```diff
@@ -3689,6 +3689,8 @@ TR_Debug::getRuntimeHelperName(int32_t index)
 
          case TR_newObject:                 return "jitNewObject";
          case TR_newObjectNoZeroInit:       return "jitNewObjectNoZeroInit";
+         case TR_newValue:                 return "jitNewObject";
+         case TR_newValueNoZeroInit:       return "jitNewObjectNoZeroInit";
```
Comment (Leonardo2718) [inline compiler/ras/Debug.cpp]: These returns are not aligned with the others (I think there's just one space missing).

Comment (tajila) [reply compiler/ras/Debug.cpp]: fixed

**Comment by fjeremic** [comment]
@genie-omr build all

**Comment by fjeremic** [comment]
@genie-omr build aix

**Pull request merged**